### ISSUE TITLE
Big and inconvenient configuration stored in environment variable

### ISSUE THREAD

**Issue by cleac** [GDG-Ukraine/gdg.org.ua#81]
### Problem

This:
```
[gdg.org.ua]$ export BLUEBERRYPY_CONFIG='{ "global": { "key":"<32-byte-str-for-aes>", "google_oauth": { "id": "<google_app_id>", "secret": "<google_app_secret>" }, "alembic": {"sqlalchemy.url": "mysql+mysqlconnector://<username>:<userpassword>@/<dbname>?unix_socket=/var/run/mysqld/mysqld.sock"} }, "sqlalchemy_engine": { "url": "mysql+mysqlconnector://<username>:<userpassword>@/<dbname>?unix_socket=/var/run/mysqld/mysqld.sock" } }'
```
looks really awful and inconvenient for usage. It is hardly understandable for those, who wants to start development environment on his own machine (like me)

### Solve

I think that good idea is to use some configuration file written in same JSON, but well formatted. This file should be loaded automatically when starting a

**Comment by anxolerd** [comment]
@cleac: you can edit `config/dev/app.yml`. You can find there sections for editing
 - [google_oauth](https://github.com/GDG-Ukraine/gdg.org.ua/blob/master/config/dev/app.yml#L9-L11)
- [sqlalchemy and alembic db url](https://github.com/GDG-Ukraine/gdg.org.ua/blob/master/config/dev/app.yml#L48)

But be careful not to push this file into repository. In order not to edit any files you can use store your credentials in environmental variable, like it is described in `README.md`

**Comment by cleac** [comment]
@anxolerd: now I see it. 
Maybe, another thing that can be done with this is a little script that asks all of these parameters automatically, with saving it into ignored by git file (add it into .gitignore)

**Comment by webknjaz** [comment]
This is not django with some tricky `local_settings.py`.
It's easy to set the variables beforehand and put in `.zshrc`/activation script/whatever.

I've implemented it to avoid editing of ignored files and putting credentials there. This dict just patches existing config with new values.

Nevertheless, this code is not related to this repo. It's stored in `blueberrypy-wk` project on bitbucket.
Closing.
